lib/card-games/analyzer.py

```diff
@@ -0,0 +1,59 @@
+"""
+Basic analyzer for the `card-games` exercise.
+Only has pylint checks active currently.
```

Comment:
```suggestion
Only has pylint checks are active currently.
```

Reply:
Would counter suggest "Only pylint checks are active currently."  As the `has` doesn't make sense.

Reply:
```suggestion
Only Pylint checks are active currently.
```